### PR TITLE
fix: update bowser

### PR DESCRIPTION
**Pull request by mwegman**
[SPARK-224282](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-224282) #done

### CODE REVIEW

**Comment by taymoork2** [comment]
This is PR is a real one

https://media.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif

File: packages/node_modules/@webex/plugin-meetings/src/metrics/index.js [hunk 1]
```diff
@@ -4,15 +4,34 @@
...
 import {MEETING_ERRORS} from '../constants';
 import StaticConfig from '../common/config';
+import BrowserDetection from '../common/browser-detection';
+
+import {
+  error, eventType, errorCodes as ERROR_CODE, OS_NAME, UNKNOWN, CLIENT_NAME
+} from './config';
+
+const OSMap = {
+  'Chrome OS': OS_NAME.chrome,
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/metrics/index.js:20]: nice

**Review by arun3528 — APPROVED** [review]
Looks good thanks @mwegman , can we test the chrome OS and see if we can dial a meeting and see the data in the control hub

**Review by lalli-flores — APPROVED** [review]
Nice! Thanks @mwegman

**Pull request merged**